kitchen.dokken.yml

```diff
@@ -1,58 +1,60 @@
----
 driver:
   name: dokken
-  # because Docker and SystemD/Upstart
-  privileged: true
-  userns_host: true
-  chef_version: <%= ENV['CHEF_VERSION'] || 'stable' %>
+  privileged: true  # because Docker and SystemD/Upstart
+  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
+  chef_license: accept-no-persist
 
-provisioner:
+transport:
   name: dokken
-  clean_dokken_sandbox: false
 
-transport:
+provisioner:
   name: dokken
+  deprecations_as_errors: true
+  chef_license: accept-no-persist
+
+verifier:
+  name: inspec
 
 platforms:
-  - name: debian-10
+  - name: amazonlinux-2
     driver:
-      image: dokken/debian-10
-      pid_one_command: /bin/systemd
-      volumes:
-        - /sys/fs/cgroup:/sys/fs/cgroup:ro # required by systemd
```

Comment:
This should be kept as this is what allows the `runas` suites to run properly on dokken.